frontend/src/app/features/statistics/statistics.component.scss

```
@@ -1,19 +1,26 @@
+@use '../../shared/styles/responsive.scss' as r;
+
 .stats-container {
+  @include r.responsive-padding;
     padding: 24px;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)